Diktat can no longer be built using Java 8

```console
$ mvn install -DskipTests=true -rf :diktat-rules

[INFO] --- exec-maven-plugin:3.1.0:java (generate-warnings) @ diktat-rules ---
[WARNING]
java.lang.UnsupportedClassVersionError: mu/KotlinLogging has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
    at java.lang.ClassLoader.defineClass1 (Native Method)
    at java.lang.ClassLoader.defineClass (ClassLoader.java:757)
    at java.security.SecureClassLoader.defineClass (SecureClassLoader.java:142)
    at java.net.URLClassLoader.defineClass (URLClassLoader.java:473)
    at java.net.URLClassLoader.access$100 (URLClassLoader.java:74)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:369)
    at java.net.URLClassLoader$1.run (URLClassLoader.java:363)
    at java.security.AccessController.doPrivileged (Native Method)
    at java.net.URLClassLoader.findClass (URLClassLoader.java:362)
    at org.codehaus.mojo.exec.URLClassLoaderBuilder$ExecJavaClassLoader.loadClass (URLClassLoaderBuilder.java:165)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:352)
    at org.cqfn.diktat.ruleset.rules.DiktatRule.<clinit> (DiktatRule.kt:80)
    at org.cqfn.diktat.ruleset.generation.GenerationKt$validateYear$2.invoke (Generation.kt:77)
    at org.cqfn.diktat.ruleset.generation.GenerationKt$validateYear$2.invoke (Generation.kt:72)
    at org.cqfn.diktat.ruleset.generation.GenerationKt.validateYear$lambda$4 (Generation.kt:72)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.accept (ForEachOps.java:183)
    at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:175)
    at java.util.Iterator.forEachRemaining (Iterator.java:116)
    at java.util.Spliterators$IteratorSpliterator.forEachRemaining (Spliterators.java:1801)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:482)
    at java.util.stream.AbstractPipeline.wrapAndCopyInto (AbstractPipeline.java:472)
    at java.util.stream.ForEachOps$ForEachOp.evaluateSequential (ForEachOps.java:150)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential (ForEachOps.java:173)
    at java.util.stream.AbstractPipeline.evaluate (AbstractPipeline.java:234)
    at java.util.stream.ReferencePipeline.forEach (ReferencePipeline.java:485)
    at org.cqfn.diktat.ruleset.generation.GenerationKt.validateYear (Generation.kt:72)
    at org.cqfn.diktat.ruleset.generation.GenerationKt.main (Generation.kt:39)
    at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:279)
    at java.lang.Thread.run (Thread.java:750)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for diktat-rules 1.2.4-SNAPSHOT:
[INFO]
[INFO] diktat-rules ....................................... FAILURE [  2.493 s]
[INFO] diktat ............................................. SKIPPED
[INFO] diktat-maven-plugin ................................ SKIPPED
[INFO] diktat-gradle-plugin ............................... SKIPPED
[INFO] org.cqfn.diktat.diktat-gradle-plugin.gradle.plugin . SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.639 s
[INFO] Finished at: 2022-11-01T16:40:36+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.1.0:java (generate-warnings) on project diktat-rules: An exception occurred while executing the Java class. mu/KotlinLogging has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0 -> [Help 1]
```

Most probably caused by #1529.

--- a/pom.xml
+++ b/pom.xml
@@ -59,7 +59,7 @@
         <jacoco.version>0.8.8</jacoco.version>
         <maven-plugin-tools.version>3.6.4</maven-plugin-tools.version>
         <jbool.version>1.24</jbool.version>
-        <mu-logging.version>3.0.3</mu-logging.version>
+        <mu-logging.version>3.0.4</mu-logging.version>
         <log4j.version>2.19.0</log4j.version>
     </properties>
 

